app/src/androidTest/java/net/skyscanner/backpack/dialog/BpkDialogTest.kt

```diff
@@ -43,8 +42,8 @@ class BpkDialogTest : BpkSnapshotTest() {
   private lateinit var activity: AppCompatActivity
 
   @get:Rule
-  var activityRule: ActivityTestRule<MainActivity> =
-    ActivityTestRule(MainActivity::class.java)
+  var activityRule: ActivityTestRule<AppCompatActivity> =
```

Comment:
TestActivity, perhaps?

Reply:
This is in the app module, not the backpack module, so the `TestActivity` doesn't exist here